orchestration/runner.go

```diff
@@ -172,20 +176,27 @@ func (r *Runner) runContainer(logger Logger, vd *variantData) error {
 	return nil
 }
 
-func listServices(servicesFile string) ([]string, error) {
+func listServices(servicesFile string) ([]commons.Service, error) {
```

Comment:
Not strictly related to your work, but I think it would be better to move this part to `orchestration/parser.go` in the `variantsData` method  where we already collect the other variant's files (`Dockerfile`, shell scripts, ..).

While we're at it, can't we simply use yaml for the services file instead of mucking around with Scanner and string splitting ?